### DISCUSSION TITLE
set to default on windows 11

### DISCUSSION THREAD

**Discussion by Ar57m** [k2-fsa/sherpa-onnx#1177] [General]
is it possible to set as the default tts engine on windows 11 like on android?
I downloaded this one sherpa-onnx-1.10.19-win-x64-en-tts-vits-piper-en_US-libritts-high.tar

**Comment by csukuangfj** [comment]
> is it possible to set as the default tts engine on windows 11 like on android?


Do you have any example to do that?

**Reply by Ar57m** [reply]
Installed windows 11 recently, I don't have much experience with it, I mean to appear like here in the screenshot, but I guess it's not possible?
![Screenshot 2024-07-29 192253](https://github.com/user-attachments/assets/3c209fe4-3432-4d3c-b593-2e2f5f5afd2c)

**Comment by Ar57m** [comment]
hey @csukuangfj sorry for the ping, recently I saw this code https://github.com/RVC-Boss/GPT-SoVITS, just a suggestion if possible to bring support. Ive runned on termux, the GPT-SoVITS v2 tts uses a bunch of models, it can kinda clone voice with less than 10s of audio, it's pretty decent, but because of missing quantizing options it was crashing a lot(the issue was RAM I think).

**Reply by csukuangfj** [reply]
sorry, there is no plan.
